cmd/ci-operator/main.go

```diff
@@ -268,7 +271,20 @@ func (o *options) Complete() error {
 		return fmt.Errorf("failed to load configuration: %v", err)
 	}
 	o.configSpec = config
-
+	if o.stepLibraryPath != "" {
+		data, err := ioutil.ReadFile(o.stepLibraryPath)
+		if err != nil {
+			return fmt.Errorf("failed to read step library (%v)", err)
+		}
+		var l []api.TestStep
+		if err := yaml.Unmarshal(data, &l); err != nil {
+			return fmt.Errorf("failed to load step library (%v)", err)
+		}
+		o.stepLibrary = make(map[string]api.TestStep, len(l))
```

Comment:
If you want this data structure, lets just serialize it instead of having to do post-processing. What happens if two names collide?

Reply:
I agreed initially, but then realized we would need code here anyway to validate that the key and the `name` field match (or similar treatment) and that duplicates would be silently ignored. Loading a list and validating that names are not duplicated seems better.